source/common/upstream/upstream_impl.h

```diff
@@ -35,6 +35,20 @@
 namespace Envoy {
 namespace Upstream {
 
+// Wrapper around envoy::api::v2::Locality to make it easier to compare and construct literals (for
```

Comment:
nit: this is used in not test code right? (Per my other comment). Change comment?

Reply:
This is used in both `eds.cc` and `eds_test.cc`. I'll update the comment.